packages/time/src/timeMath.js

```diff
@@ -200,7 +208,7 @@ const modRelRel = (rel, step) =>
  * @param {Timestamp | RelativeTime} right
  * @param {bigint} v1
  * @param {bigint} v2
- * @returns {RankComparison}
+ * @returns {-1 | 0 | 1}
```

Comment:
Why this change?

Reply:
because the type wasn't resolving. since TS is structurally typed it is functionally no different.

Reply:
You understand vastly more than I about why the type might not be resolving, or how to get it to resolve, than I do. If this can be repaired at reasonable code, I'd prefer that, since `{RankComparison}` communicates intent to future readers that `{-1 | 0 | 1}` does not. But if that's hard, the second is fine with a comment. (I'm much more concerned with human understandability than type-checking equivalence.)